### PR TITLE
INCR-94: Support testing Python 3 in django-cas

### PR DESCRIPTION
**Pull request by cclauss**
Fixes: https://openedx.atlassian.net/browse/INCR-94

Travis-CI needs to be enabled for this repo.  https://travis-ci.org/mitodl/django-cas

### CODE REVIEW

**Comment by jmbowman** [comment]
@pdpinch Do you have anyone with time to review this and/or enable CI for this repo?

**Comment by estute** [comment]
@pdpinch just wanted to follow up on this. Do you think you would be able to review/help out with this PR?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - "mkdir var"
+    - "wget https://github.com/mozilla/geckodriver/releases/download/v0.15.0/geckodriver-v0.15.0-linux64.tar.gz"
+    - "mkdir geckodriver"
+    - "tar -xzf geckodriver-v0.15.0-linux64.tar.gz -C geckodriver"
+    - "export PATH=$PATH:$PWD/geckodriver"
+    - "pip install -r requirements/dev.txt"
+    - "pip install -r requirements/test.txt"
+script:
+    - tox -e $TOXENV
+deploy:
```
Comment (estute) [inline .travis.yml]: @cclauss deploying to pypi is beyond the scope of this INCR ticket. Could you remove this section?

**Comment by blarghmatey** [comment]
@cclauss I enabled Travis on this repo if you want to make an empty commit to trigger the build.

**Pull request merged**